Automattic/wp-calypso
# Stepper: update 'InlineSupportLink' link component

#### Changes proposed in this Pull Request

* Updated `InlineSupportLink` component to support switching between opening support content in modal or directly redirecting to the support link in a new tab.
* Before the change, in the Stepper framework support links don't work because of `page.js` lib dependency.
* Added `ImportDrag` config file: remove redundant config logic

#### Testing instructions

* Go to `/setup/import?siteSlug={SLUG}`
* Select `I don't have a site address` option in the top-right corner
* Pick any of the first four importers
* Check if the `Need help exporting your content?` support link opens a link in the new tab

<img width="798" alt="Screenshot 2022-05-17 at 17 34 57" src="https://user-images.githubusercontent.com/1241413/168859809-77d2bad1-caf9-402b-8096-1d54cd181a86.png">



Related to #62238
